### PR TITLE
Migration provided metrics.adoc small fixes

### PR DESCRIPTION
**Pull request by sbeskin-redhat**
migration-provided-metrics.adoc - small fixes 

4.6+

Preview: 
https://deploy-preview-38554--osdocs.netlify.app/openshift-enterprise/latest/migrating_from_ocp_3_to_4/troubleshooting-3-4.html#migration-provided-metrics_troubleshooting-3-4

### CODE REVIEW

File: modules/migration-provided-metrics.adoc [hunk 1]
```diff
@@ -38,7 +38,7 @@ This metric is a cumulative count of Kubernetes API requests that `MigrationCont
 |Queryable label name |Sample label values |Label description
 
 |cluster
-|`\https://migcluster-url:443`
+|`\https://www.migcluster.com:443`
```
Comment (apinnick) [inline modules/migration-provided-metrics.adoc:41]: See OpenShift guidelines: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#example-urls

Comment (sbeskin-redhat) [reply modules/migration-provided-metrics.adoc:41]: @apinnick 
Do you mean that it should be literally \https://www.example.com? I thought the word 'example' should be replaced with something meaningful.

Comment (apinnick) [reply modules/migration-provided-metrics.adoc:41]: Yes. "migcluster" means nothing to the end user. You can't make it any more meaningful.

**Comment by apinnick** [comment]
@stoobie @sbeskin-redhat @nunzy1 

Please review this PR and decide what should be done with it.

**Comment by stoobie** [comment]
> @stoobie @sbeskin-redhat @nunzy1
> 
> Please review this PR and decide what should be done with it.

@apinnick Thank you for the heads-up. We'll follow up

**Comment by nalhadef** [comment]
The content looks fine, but there are five commits. Let me know when they've been squashed and I'll add the LGTM label.

File: modules/migration-provided-metrics.adoc [hunk 2]
```diff
@@ -38,7 +38,7 @@ This metric is a cumulative count of Kubernetes API requests that `MigrationCont
 |Queryable label name |Sample label values |Label description
```
Comment (apinnick) [inline modules/migration-provided-metrics.adoc:38]: Module needs a content-type attribute.

**Review by apinnick — CHANGES_REQUESTED** [review]

**Comment by sbeskin-redhat** [comment]
@nalhadef Done. Thank you!

**Comment by sbeskin-redhat** [comment]
@nalhadef  Thank you! Will you merge it?

**Pull request closed without merging**